lib/can-npm-publish.js

```diff
@@ -31,6 +31,36 @@ const checkPrivateField = packagePath => {
     });
 };
 
+/**
+ * Return Promise which resolves with an array of version numbers for the package
+ * or rejects if anything failed
+ * @param packageName
+ * @returns {Promise}
+ */
+const viewPackage = packageName => {
+    return new Promise((resolve, reject) => {
+        const view = spawn("npm", ["view", packageName, "versions", "--json"]);
+        let result = "";
+
+        view.stdout.on("data", data => {
+            result += data.toString();
+        });
+
+        view.stderr.on("data", err => {
+            console.error(err.toString());
+        });
+
+        view.on("close", code => {
+            if (code > 0) {
+                reject();
```

Comment:
```js
reject(new Error(errorResult));
```

We should reject with `errorResult`, because`reject` should have Error arguments.
This error message will be output to https://github.com/azu/can-npm-publish/blob/e8ea9377adf5f6d4773a98b278e09a33105e1bea/bin/cmd.js#L39.